dotCMS/src/main/webapp/html/portlet/ext/contentlet/publishing/view_unpushed_bundles.jsp

```diff
@@ -90,15 +90,16 @@
 						<th width="100%" onclick="goToEditBundle('<%=bundle.getId()%>')" style="cursor:pointer">
 							<b><%=StringEscapeUtils.unescapeJava(bundle.getName())%></b> 
                             (<span> <%=bundle.getId() %> </span>)
+							<span> ( <%=APILocator.getUserAPI().loadUserById(bundle.getOwner()).getFullName()%> ) </span>
```

Comment:
Please check if the User exists first. There might be cases in which it doesn't and this may thrown an NPE

Reply:
Thanks, done! https://github.com/dotCMS/core/pull/27371/commits/e589b80f8d3d32ae24adcebf300624fc3f9e6c9e